includes/data-port/import-tasks/class-sensei-import-file-process-task.php

```diff
@@ -75,7 +75,23 @@ public function __construct( Sensei_Data_Port_Job $job ) {
 			$task_state      = $this->get_job()->get_state( $this->get_task_key() );
 			$completed_lines = isset( $task_state[ self::STATE_COMPLETED_LINES ] ) ? $task_state[ self::STATE_COMPLETED_LINES ] : 0;
 
-			$this->reader             = new Sensei_Import_CSV_Reader( get_attached_file( $attachment_id ), $completed_lines );
+			try {
+				$this->reader = new Sensei_Import_CSV_Reader( get_attached_file( $attachment_id ), $completed_lines );
+			} catch ( Exception $e ) {
+				$this->get_job()->add_log_entry(
+					__( 'Uploaded file could not be opened while continuing.', 'sensei-lms' ),
```

Comment:
Could be simply `'Uploaded file could not be opened.'`
(Even so because this might happen on the first try too)

Reply:
I started with that, but I wanted to create a name to be different from here `Sensei_Import_CSV_Reader::validate_csv_file`, so if a user report that, we could identify better where the error happened. But I changed for now, and if we think we should use a different message, we can change later. 😊

https://github.com/Automattic/sensei/pull/3484/commits/89d2d9cc0403d456081bb50031a7921ba0c9df9a